src/SDK/MonitoringApplianceEmulator/ViewModels/ChartPropertyControlViewModel.cs

```diff
@@ -160,25 +175,35 @@ private void InitializeNumberXAxisChart(ChartAlertProperty chartAlertProperty)
         {
             var sortedNumericPoints = chartAlertProperty.DataPoints.OrderBy(point => point.X).ToList();
 
-            var chartDatePoints = sortedNumericPoints.Select(dataPoint =>
+            var chartPoints = sortedNumericPoints.Select(dataPoint =>
             {
                 double x = ConvertCoordinateValueToDouble(dataPoint.X, "X");
                 double y = ConvertCoordinateValueToDouble(dataPoint.Y, "Y");
 
                 return new ChartDataPoint<double>(x, y);
             });
 
-            var chartValues = new ChartValues<ChartDataPoint<double>>(chartDatePoints);
+            var chartValues = new ChartValues<ChartDataPoint<double>>(chartPoints);
 
             CartesianMapper<ChartDataPoint<double>> pointMapperConfig = Mappers.Xy<ChartDataPoint<double>>()
-                .X(dateTimeDataPoint => dateTimeDataPoint.X)
-                .Y(dateTimeDataPoint => dateTimeDataPoint.Y);
+                .X(dataPoint => dataPoint.X)
+                .Y(dataPoint => dataPoint.Y);
 
             this.SeriesCollection = new SeriesCollection(pointMapperConfig);
 
             this.AddSeries(chartAlertProperty.ChartType, chartValues);
 
             this.XAxisFormatter = value => value.ToString(CultureInfo.InvariantCulture);
+
+            // Set the Y axis format
+            if (chartAlertProperty.YAxisType == ChartAxisType.Percentage)
+            {
+                this.YAxisFormatter = value => $"{value.ToString(CultureInfo.InvariantCulture)}%";
+            }
+            else
+            {
+                this.YAxisFormatter = value => value.ToString(CultureInfo.InvariantCulture);
+            }
```

Comment:
since it is exactly the same logic as above, you can extract it to a method that returns that formatter func